opt/scopeserver/__init__.py

```diff
@@ -1,19 +1,20 @@
-from scopeserver.dataserver.modules.gserver import GServer as gs
-from scopeserver.dataserver.modules.pserver import PServer as ps
-from scopeserver.bindserver import XServer as xs
-from scopeserver.dataserver.utils import sys_utils as su
-
+import argparse
 import threading
-import os
 import time
 from urllib.request import urlopen
 import http
 import sys
 import logging
-import json
 import secrets
+from pathlib import Path
 from typing import Dict, Any
 
+from scopeserver.dataserver.modules.gserver import GServer as gs
+from scopeserver.dataserver.modules.pserver import PServer as ps
+from scopeserver.bindserver import XServer as xs
+from scopeserver.dataserver.utils import sys_utils as su
+from scopeserver.scope.config import from_file
+
 LOG_FMT = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
 logging.basicConfig(format=LOG_FMT, level=logging.INFO)
 logger = logging.getLogger(__name__)
```

Comment:
`logger` should be changes to `LOGGER` to keep consistency with the other changes here.

Reply:
Done. Thanks